mlir/include/mlir/Dialect/EmitC/Transforms/Passes.td

```diff
@@ -21,4 +21,15 @@ def FormExpressions : Pass<"form-expressions"> {
   let dependentDialects = ["emitc::EmitCDialect"];
 }
 
+def EliminateLibm : Pass<"eliminate-libm"> {
+  let summary = "Replace Libm function declarations by a math.h inclusion";
+  let description = [{
+    This pass removes of the Libm function declarations (marked by the
+    `libm` attribute) and replaces them by a `<math.h>` inclusion.
+    Calls to these functions are replaced by `emitc.opaque` calls.
+  }];
+  let constructor = "mlir::emitc::createEliminateLibmPass()";
```

Comment:
```suggestion
```
and remove the definition of this function from EliminateLibm.cpp. It will then be automatically generated.